packages/x-components/tests/e2e/cucumber/search-box/search-box.spec.ts

```diff
@@ -157,7 +158,7 @@ And('related tags are displayed after instantDebounceInMs is {boolean}', (instan
 // Scenario 4
 Given('a second results API with a known response', () => {
   cy.intercept('https://api.empathy.co/search', req => {
-    req.reply({
+    req.reply(<SearchResponse>{
```

Comment:
Uhm, is it working for you? I have to add the missing parts in the response so the editor doesn't 'yell'
```
req.reply(<SearchResponse>{
      banners: [],
      promoteds: [],
      facets: [],
      results: [
        createResultStub('LEGO Duplo Disney Tren de Cumpleaños de Mickey y Minnie - 10941', {
          images: ['https://picsum.photos/seed/8/100/100']
        }),
        createResultStub('LEGO Disney Granja de Mickey Mouse y el Pato Donald - 10775', {
          images: ['https://picsum.photos/seed/10/100/100']
        })
      ],
      redirections: [],
      partialResults: [],
      totalResults: 7,
      queryTagging: {
        url: 'https://tagging.empathy.co',
        params: {}
      },
      spellcheck: 'lego'
    });
```

Reply:
You are right, I forgot about this one.